src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -55,7 +55,6 @@ module sonic-port{
 				}
 
 				leaf speed {
-					mandatory true;
```

Comment:
Can you please explain how does speed get populated to APP_DB if it is not part of config_db? In SAI port speed is a mandatory parameter. Will it be taken from platform files? If yes can you please point to the code?